src/prefect/run_configs/kubernetes.py

```diff
@@ -108,6 +121,16 @@ def __init__(
         if image_pull_secrets is not None:
             image_pull_secrets = list(image_pull_secrets)
 
+        image_pull_policies = {"Always", "IfNotPresent", "Never"}
+        if (
+            image_pull_policy is not None
+            and image_pull_policy not in image_pull_policies
+        ):
+            raise ValueError(
+                f"Invalid image_pull_policy {image_pull_policy}.  "
+                "Expected `Always`, `IfNotPresent`, or `Never`"
```

Comment:
```suggestion
                "Expected 'Always', 'IfNotPresent', or 'Never'"
```
We expect string values whereas ` implies an object to me